dashboard/client/src/components/FunctionTableItem/FunctionTableItem.jsx

```diff
@@ -17,6 +17,69 @@ const genRepoUrl = ({ gitOwner, gitRepoURL }) => (
   `${gitRepoURL}/commits/master`
 );
 
+const genOwnerInitials = (owner) => (
```

Comment:
This looks really cryptic. Could it be written another way?

Reply:
Agree with cryptic, unit tests would help here. Also appears quite a fragile structure. Is there another way to get this information?

Reply:
The complication comes because we want if the name contains `-` or `_` initials to be formed by the first letters of each part, otherwise to be formed by the first two letters. Unless there is a built in way to do this, we cannot escape from the regex and check.
@jagreehal if you know some existing approach that can replace the logic here, can you please share?

Would documenting work?